homeassistant/components/roborock/coordinator.py

```diff
@@ -107,3 +109,10 @@ def _set_current_map(self) -> None:
             self.current_map = (
                 self.roborock_device_info.props.status.map_status - 3
             ) // 4
+
+    async def get_maps(self) -> None:
```

Comment:
Why is this method only called ones in the init and not on an update?
If you only need to call it on setup, why are you placing this function in the coordinator? You can pass the returned data directly to `create_coordinator_maps`

Reply:
> * The ability to change maps
> 
> * Room selection. Room ids are map specific, so we need to be able to map from the map to the room to be able to accurate perform tasks on it. So the coordinator (Or my RoborockHassDeviceInfo model) would need the map information

They will be implemented in a follow-up, correct?

> Why is this method only called ones in the init and not on an update?

Can you please answer this question

Reply:
> They will be implemented in a follow-up, correct?
Yep! 
> Can you please answer this question

Sorry missed that part. It could be run on every update but it's a lot less likely to be changed. Only when a new map is created. Which would also mean a new entity would be needed - which isn't currently supported. So my thought was if someone adds a new map, they need to reload anyways. I can make it run every update though if you think that would be good

Reply:
Sorry this month we had a lot of contributions, and I missed your PR.

No it's fine. I was only wondering why it's only called ones and it is part of the coordinator

Reply:
No worries. I know there are a ton of PRs and always a lot going on. Thanks for the review/ approval